packages/common/src/ee/AiAgent/index.ts

```diff
@@ -50,6 +50,7 @@ export const baseAgentSchema = z.object({
         .nullable(),
     provider: z.string(),
     model: z.string(),
+    groupAccess: z.array(z.string()).nullable(),
```

Comment:
This is always going to be set going forward right? If there's no group access setup it's going to be an empty array

Reply:
Should `nullable()` be removed then?